roles/ceph-mon/tasks/docker/main.yml

```diff
@@ -6,6 +6,9 @@
 - name: include start_docker_monitor.yml
   include: start_docker_monitor.yml
 
+- name: include drop_ceph_command_aliases.yml
```

Comment:
Can we rename this simply `configure_ceph_command_aliases.yml`? The term 'drop' can be misleading.